data/projects.js

```diff
@@ -147,5 +147,119 @@ coala should support generating metrics for your code.",
 		"initiatives" : ["GSoC"],
 		"tags" : ["Web", "Blog"],
 		"markdown": "coala_web.md"
+	},
+	{
+		"name" : "Optimize Caching",
```

Comment:
took the project from the project proposal page. I feel like we should make smaler projects than last year so they get merged. Not a lot from last year was merged :/